### PR TITLE
639 openstack managed page

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update `openstack/managed`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/openstack/managed>
- Check that the page matches the [copy doc](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#)

## Issue / Card

Fixes [#639](https://github.com/ubuntudesign/web-squad/issues/639)

### CODE REVIEW

**Comment by anasereijo** [comment]
Hi @steverydz see bellow some changes:

screen shot 2018-06-20 at 15 45 24

screen shot 2018-06-20 at 16 20 57

(copy doc for this one is updated)
screen shot 2018-06-20 at 16 58 26

screen shot 2018-06-20 at 17 02 49

**Comment by anasereijo** [comment]
@steverydz just 2 more things and it's ready to go:

- this looks the same. can you please make it look like the following image?
41670498-5c7120c0-74ac-11e8-82ef-dd29574923cf

- make the following sections light grey:

screen shot 2018-06-21 at 14 01 18
screen shot 2018-06-21 at 14 02 25
screen shot 2018-06-21 at 14 02 36

**Comment by steverydz** [comment]
@anasereijo Changes made

**Comment by frankiegranato** [comment]
@anasereijo Can you take one more look at the OpenStack logo? Its a bit floaty.

**Comment by frankiegranato** [comment]
- [ ] Em dash replaces a hyphen

**Comment by frankiegranato** [comment]
451 card
@anasereijo Can you have a quick look at the 451 card again? It looks a bit weird.

**Comment by anasereijo** [comment]
@steverydz changes below:

screencapture-ubuntu-pr-3383-run-demo-haus-openstack-managed-2018-06-22-10_54_19

**Comment by steverydz** [comment]
@anasereijo @frankiegranato Done

**Comment by anasereijo** [comment]
@steverydz looking good! just make the following background white and it's done

screen shot 2018-06-22 at 15 05 44

thanks

**Comment by steverydz** [comment]
@anasereijo Done

**Comment by pmahnke** [comment]
I think the buttons in the top banner are backwards (white then green) I would just move their position (talk to us second).

**Comment by steverydz** [comment]
@pmahnke Done

**Comment by frankiegranato** [comment]
Looks great. One small change (hyphen to m-dash), but aside form that its a +1 from me.

**Comment by steverydz** [comment]
@frankiegranato Done

File: templates/openstack/managed.html [hunk 1]
```diff
@@ -5,63 +5,114 @@
...
     </div>
   </div>
 </section>
 
 <section class="p-strip is-deep is-bordered">
   <div class="row">
     <div class="col-12">
-      <h2>Economics, performance and reliability</h2>
+      <h2>Performance and reliability at scale</h2>
+      <p>Cloud infrastructure needs to be competitive with public cloud, and it needs to be fast and reliable. Our experience with OpenStack at scale ensures that we deliver efficient and resilient clouds - with an SLA.</p>
```
Comment (anthonydillon) [inline templates/openstack/managed.html]: Please convert the `-` to `&mdash;`

File: templates/openstack/managed.html [hunk 2]
```diff
@@ -96,12 +147,18 @@ <h3 class="p-heading--four"><a class="p-link--external" href="https://pages.ubun
 
 
 <section class="p-strip--light is-deep is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <h2>Security trusted by the OpenStack majority</h2>
```
Comment (anthonydillon) [inline templates/openstack/managed.html]: There is a `mdash` in the copy doc

File: templates/openstack/managed.html [hunk 3]
```diff
@@ -114,60 +171,66 @@ <h2>Security trusted by the OpenStack majority</h2>
...
+        <li class="p-inline-images__item">
+          <img src="{{ ASSET_SERVER_URL }}d0b7aa28-logo-deutsche-telekom.png" alt="Deutsche Telekom logo" />
         </li>
         <li class="p-inline-images__item">
-          <img src="{{ ASSET_SERVER_URL }}66bc9135-logo-lexisnexis.png" alt="LexisNexis logo" />
+          <img src="{{ ASSET_SERVER_URL }}481063cd-pccw.svg" alt="PCCW logo" />
         </li>
         <li class="p-inline-images__item">
-          <img src="{{ ASSET_SERVER_URL }}25de1877-Tele2.svg" alt="Tele2 logo" />
+          <img src="{{ ASSET_SERVER_URL }}7dfdecc2-cogeco-logo_colour.png" alt="Cogeco Datacenters logo" />
```
Comment (anthonydillon) [inline templates/openstack/managed.html]: This logo doesnt work on FF

File: templates/openstack/managed.html [hunk 4]
```diff
@@ -381,40 +493,42 @@ <h2>Shape it to your needs</h2>
...
         </div>
       </li>
       <li class="p-matrix__item">
         <div class="p-matrix__content">
-          <h3 class="p-matrix__title">Build services</h3>
-          <p class="p-matrix__desc">We can build your cloud. Get started fast with our OpenStack reference architecture or a custom design to optimise your workloads.</p>
-          <p class="p-matrix__desc"><a href="/openstack/consulting">OpenStack build service&nbsp;&rsaquo;</a></p>
+          <h3 class="p-matrix__title">Cloud build services</h3>
+          <p>Even if you want to operate the cloud from the beginning, we can still help you design and build it.</p>
+          <p>Get started with our <a href="/openstack/consulting">reference architecture</a> to optimise your workloads.</p>
```
Comment (anthonydillon) [inline templates/openstack/managed.html]: Get started **fast** with our reference architecture to optimise your workloads.

**Comment by steverydz** [comment]
@anthonydillon All fixed

**Pull request merged**